toolbox/library/include/WBToolbox/Block/OSQP.h

```diff
@@ -58,8 +61,7 @@ class wbt::block::OSQP final : public blockfactory::core::Block
     bool parseParameters(blockfactory::core::BlockInformation* blockInfo) override;
     bool configureSizeAndPorts(blockfactory::core::BlockInformation* blockInfo) override;
     bool initialize(blockfactory::core::BlockInformation* blockInfo) override;
-    bool
-    initializeInitialConditions(const blockfactory::core::BlockInformation* blockInfo) override;
+    bool initializeInitialConditions(const blockfactory::core::BlockInformation* blockInfo) override;
```

Comment:
please run clang-format before merging (refer to https://github.com/robotology/wb-toolbox/blob/master/.github/CONTRIBUTING.md#code-style), should also solve an old issue https://github.com/robotology/wb-toolbox/issues/216

Reply:
Done